udata-front/package.json

```diff
@@ -0,0 +1,21 @@
+{
+  "name": "udata-front",
+  "version": "1.0.0",
+  "description": "Open Data Portal - Front-end library",
+  "author": "Etalab",
+  "main": "dist/index.js",
+  "files": ["dist", "src"],
+  "scripts": {
+    "clean": "rimraf dist",
+    "build": "tsc",
+    "prebuild": "npm run clean",
+    "preversion": "npm run build",
+    "postversion": "git push --follow-tags"
+  },
+  "devDependencies": {
+    "rimraf": "^3.0.2",
```

Comment:
Do we need this?

Reply:
It's used to clear the `dist` folder before publishing a new version.

Due to the size of the package, we can remove it because we're only building a single file and the generated files are always the same.